dispatcher/DAQController.py

```diff
@@ -277,8 +277,9 @@ def check_timeouts(self, detector, command=None):
                 self.hypervisor.tactical_nuclear_option()
                 self.last_nuke = now()
             else:
+                #self.control_detector(detector=detector, command='stop')
```

Comment:
would we need to check that the command is `stop` to prevent line 284 from trying to issue another command which is not stop?

Reply:
That's why I commented this line out. I considered it but wasn't sufficiently sure it was what we wanted.

Reply:
then, perhaps logging the current command makes sense?

Reply:
If we get to here then the readout has not been able to arm successfully, and hasn't yet returned to idle after the last nuke. Whatever the last command issued was, it didn't have the desired effect. I commented the line out because think that just commenting the `return` will accomplish the desired outcome because then we go through the rest of the logic in this function.